### ISSUE TITLE
possible leak in MinimalLib?

### ISSUE THREAD

**Issue by eloyfelix** [eloyfelix/RDKitMinimalLib.jl#12]
RDKitMinimalLib.jl gets memory hungry when iterating:

```julia
using RDKitMinimalLib

for i in 1:1000000
    mol = get_mol("CC(=O)Oc1ccccc1C(=O)O")
end
```

same happens when running a minimal example calling directly RDKit_jll:

```julia
using RDKit_jll

for i in 1:1000000
    mol_size = Ref{Csize_t}(0)
    mol::Ptr{UInt8} = ccall((:get_mol, librdkitcffi), Ptr{UInt8}, (Ptr{UInt8}, Ref{Csize_t}, Ptr{UInt8}), "CC(=O)Oc1ccccc1C(=O)O", mol_size, "")
    ccall((:free_ptr, librdkitcffi), Cvoid, (Ptr{UInt8},), mol)
end
```

Also happening with Python ctypes using RDKit example: https://github.com/rdkit/rdkit/issues/4102

**Comment by eloyfelix** [comment]
confirmed as a RDKit MinimalLib leak

**Comment by eloyfelix** [comment]
fixed in RDKit and PR sent to Yggdrasil https://github.com/JuliaPackaging/Yggdrasil/pull/2939

**Comment by eloyfelix** [comment]
fixed in RDKit_jll v2021.9.1-pre+1
